src/display/editor/ink.js

```diff
@@ -468,6 +478,84 @@ class InkEditor extends AnnotationEditor {
     this.addCommands({ cmd, undo, mustExec: true });
   }
 
+  #drawPoints() {
+    const thickness = Math.ceil(this.thickness * this.parentScale);
+    const lastPoints = this.currentPath.slice(-3);
+    const x = lastPoints.map(xy => xy[0]);
+    const y = lastPoints.map(xy => xy[1]);
+    const xMin = Math.min(...x) - thickness;
+    const xMax = Math.max(...x) + thickness;
+    const yMin = Math.min(...y) - thickness;
+    const yMax = Math.max(...y) + thickness;
+
+    const { ctx } = this;
+    ctx.save();
+    ctx.clearRect(xMin, yMin, xMax - xMin, yMax - yMin);
+    ctx.beginPath();
+    ctx.rect(xMin, yMin, xMax - xMin, yMax - yMin);
+    ctx.clip();
+
+    for (const path of this.bezierPath2D) {
+      ctx.stroke(path);
+    }
+    ctx.stroke(this.#currentPath2D);
+
+    ctx.restore();
+  }
+
+  #makeBezierCurve(path2d, x0, y0, x1, y1, x2, y2) {
```

Comment:
Nit: In the existing code we're using `path2D`, note the capitalization, so can we perhaps use a consistent format through this file to prevent surprises/issues later on?